aten/src/ATen/core/ivalue.cpp

```diff
@@ -861,7 +861,7 @@ std::ostream& operator<<(std::ostream & out, const IValue & v) {
     }
 
   }
-  AT_ERROR("Tag not found: ", v.tagKind());
+  return out << "InvalidIValue(tag=" << std::to_string(static_cast<uint32_t>(v.tag)) << ")";
```

Comment:
nit: this is not consistent with other objects being printed here. But I guess that's fine.

Reply:
good point - switched it to `<Invalid IValue tag=1234>`